docs(openai): PHP OpenAI Integration (#18087)
DataDog/integrations-core: openai/README.md

[9]: https://github.com/DataDog/dd-trace-js
[3]: https://ddtrace.readthedocs.io/en/stable/integrations.html#openai

<!-- xxz tab xxx -->
<!-- xxx tab "PHP" xxx -->

> **Note**: This setup method does not collect `openai.api.usage.*` metrics. To collect these metrics, also follow the API key setup instructions.

Reviewer: Can you link to the API key setup from here? 

```suggestion
**Note**: To collect `openai.api.usage.*` metrics, follow the [API key setup instructions][X].
```

  gcr.io/datadoghq/agent:latest
```

2. [Install the Datadog APM PHP library][16].

3. The library is automatically injected into your OpenAI PHP application.

**Notes**:

<!-- partial
{{% site-region region="us3,us5,eu,gov,ap1" %}}
- Non-US1 customers must set `DD_SITE` on the application command to the correct Datadog site parameter as specified in the table in the <a href="https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site">Datadog Site</a> page (for example, `datadoghq.eu` for EU1 customers).{{% /site-region %}}
partial -->

- If the Agent is using a non-default hostname or port, be sure to also set `DD_AGENT_HOST`, `DD_TRACE_AGENT_PORT`, or `DD_DOGSTATSD_PORT`.

See the [APM PHP library documentation][17] for more advanced usage.

### Configuration

See the [APM PHP library documentation][17] for all the available configuration options.

#### (beta) Log Prompt & Completion Sampling

To enable log prompt and completion sampling, set the `DD_OPENAI_LOGS_ENABLED="true"` environment variable. By default, 10% of traced requests will emit logs containing the prompts and completions.

To adjust the log sample rate, see the [APM library documentation][17].

> **Note**: It is recommended to enable `DD_LOGS_INJECTION` to ensure logs are correlated with traces.

Reviewer: ```suggestion
**Note**: To ensure logs are correlated with traces, Datadog recommends you enable `DD_LOGS_INJECTION`.
```

[9]: https://github.com/DataDog/dd-trace-js
[3]: https://ddtrace.readthedocs.io/en/stable/integrations.html#openai

<!-- xxz tab xxx -->
<!-- xxx tab "PHP" xxx -->

> **Note**: To collect `openai.api.usage.*` metrics, follow the [API key setup instructions][X].

Reviewer: ```suggestion
**Note**: To collect `openai.api.usage.*` metrics, follow the API key setup instructions.
```
